spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -62,7 +64,8 @@ public static void startMetastoreAndSpark() {
         .enableHiveSupport()
         .getOrCreate();
 
-    SparkTestBase.catalog = new HiveCatalog(spark.sessionState().newHadoopConf());
+    SparkTestBase.catalog = (HiveCatalog)
+        CatalogUtil.loadCatalog(HiveCatalog.class.getName(), "hive", ImmutableMap.of(), hiveConf);
```

Comment:
nit: `hiveConf` is not technically the same as `sessionState().newHadoopConf()` but it does not matter in tests.